ci/docs/layout.html

```diff
@@ -12,4 +12,36 @@
   ga('send', 'pageview');
 
 </script>
+
+{# check current version against supported-releases.json #}
+<script>
+$( document ).ready( function() {
+
+  // gets version number minus whitespace and bugfix version
+  var $version_number = $('.version').text().trim().split('.', 2).join('.')
+
+  var supported_versions = []
+  // get JSON from server and load supported_versions
+  $.getJSON("https://raw.githubusercontent.com/pulp/pulp_packaging/master/ci/config/releases/supported-releases.json" , function(data) {
+    $.each(data, function(value) {
+      supported_versions.push(value)
+    })
+  })
+
+  console.log(supported_versions)
+
+  // render warning if current version not in supported_versions
+  if ($.inArray($version_number, supported_versions) == -1) {
```

Comment:
this should be === because javascript. (it probably doesn't matter here, but it might because js sometimes does some questionable things)